### ISSUE TITLE
Multi-intent slot filling

### ISSUE THREAD

**Issue by argideritzalpea** [RasaHQ/rasa-sdk#85]
**Issue**:

Currently, there is no logic to allow multiple slot filling functions when multi-intents are registered during an active form.
 
We should be able to fill multiple slots from a predicted “multi-intent” (e.g. intent_X + intent_Y) from within a Form.

Example: 
Intent_X and and Intent_Y are registered in a FormAction, and each fill a corresponding slot and entity when they are predicted as separate intents. However, a user should be able to update multiple slots at once if such a setting is enabled to allow one to do so. 

In other words, a user should be able to fills the slots tied to both Intent_X and Intent_Y if they communicate both intents and a multi-intent X+Y is returned.

https://forum.rasa.com/t/multiple-slot-filling-from-multi-intent-intent-x-intent-y/10759

**Comment by wochinge** [comment]
Thanks for raising this issue, @Ghostvv will get back to you about it soon.

**Comment by Ghostvv** [comment]
your multi-intent X+Y should be still part of the domain, so you can override `extract_requested_slot ` https://github.com/RasaHQ/rasa-sdk/blob/b18f088446e405608b74193e84c0d82ddfe6af47/rasa_sdk/forms.py#L268 method in your form action to cover this case

**Comment by argideritzalpea** [comment]
Perhaps I can modify https://github.com/RasaHQ/rasa-sdk/blob/b18f088446e405608b74193e84c0d82ddfe6af47/rasa_sdk/forms.py#L184

to add the tracker's last message's intent; if it is a "+" multi-intent, I will add both components to the desired mapping_intents object?

**Comment by Ghostvv** [comment]
might work

**Comment by argideritzalpea** [comment]
Hmm, my proposal didn't seem to work at Line 184. In any case, it would be good to not have to define slot mappings for all combinations of "+" intents

**Comment by Ghostvv** [comment]
you can create "+" slots for "+" intents and then in validate populate appropriate slots
